src/main/kotlin/com/raeids/stratus/config/StratusConfig.kt

```diff
@@ -17,11 +17,31 @@ object StratusConfig : Vigilant(File(Stratus.modDir, "${Stratus.ID}.toml"), Stra
 
     @Property(
         type = PropertyType.SWITCH,
-        name = "Chat Searching",
-        description = "Add a chat search bar.",
-        category = "Searching"
+        name = "Custom Chat Height",
+        description = "Allows you to change the height of chat to heights greater than before.",
+        category = "Chat Window"
     )
-    var chatSearch = true
+    var customChatHeight = true
+
+    @Property(
+        type = PropertyType.SLIDER,
+        min = 180,
+        max = 10000,
```

Comment:
WeirdChamp make this like 1000 or 2000

Reply:
screens are bigger than 2000 pixels smh

Reply:
the old max is 180
you dont need to make it 10000000000 
it makes the sliders p much useless